src/levelset/levelSetSegmentationObject.cpp

```diff
@@ -2686,7 +2706,13 @@ std::array<double,3> LevelSetSegmentationObject::_evalCellNormal(long id, bool s
     long support = evalCellSupport(id);
     std::array<double,3> centroid = m_kernel->computeCellCentroid(id);
 
-    return _evalNormal(centroid, support, signedLevelSet);
+    std::array<double, 3> projectionPoint;
+    std::array<double, 3> projectionNormal;
+    _evalProjection(centroid, support, signedLevelSet, &projectionPoint, &projectionNormal);
+
+    BITPIT_UNUSED(projectionPoint);
```

Comment:
There is no need to mark projectionPoint as unused.

Reply:
I deleted it. I know it's not needed, but I thought that in this way the code is clearer.